plugins/shortcode-plugin/includes/block-editor/blocks/random-photos/index.js

```diff
@@ -39,21 +40,11 @@ registerBlockType(name, {
 				type: 'block',
 				blocks: ['core/shortcode'],
 				isMatch: ({ text }) => {
-					return text.startsWith('[twitch_example');
+					return text && text.startsWith('[twitch_example');
```

Comment:
Otherwise, causes an error when creating an empty shortcode block as `text` is `undefined`